### ISSUE TITLE
Indication of subtasks competion in the cards

### ISSUE THREAD

**Issue by ufalke** [wekan/wekan#4050]
## Issue

Hi,

I am new to Wekan, just looking for a tool to not lose track with my projects. 
Now, i added some cards in a board, and some subtasks in each card. Some of the subtasks have been completed by now, so I entered an "End Date" for them. But that does neither result in those subtasks being marked as complete in the card nor in counting them. There is a field 0/n in each card display where n is the number of subtasks, apparently. If there is one completed subtask I expect that to read 1/n. However, the first number always is 0. That is not very helpful if one wants to know the subtask status for a card at a glance.

I did not find a useful communication channel for that issue so I am trying to convey it here. 

Uwe Falke

Note: With Docker, please don't use latest tag. Only use release tags.
See https://github.com/wekan/wekan/issues/3874

If you can not login for any reason:
- https://github.com/wekan/wekan/wiki/Forgot-Password

Email settings:
- https://github.com/wekan/wekan/wiki/Troubleshooting-Mail

Add these issues to elsewhere:
- SECURITY ISSUES: https://github.com/wekan/wekan/blob/master/SECURITY.md
- UCS: https://github.com/wekan/univention/issues

Other Wekan issues can be added here.

**Server Setup Information**:

* Note: Please anonymize info, and do not add to this public issue any of your Wekan board URLs, passwords, API tokens etc, do you understand?:
* Did you test in newest Wekan?:
* For new Wekan install, did you configure root-url correctly so Wekan cards open correctly https://github.com/wekan/wekan/wiki/Settings ?
* Wekan version:
* If this is about old version of Wekan, what upgrade problem you have?:
* Operating System:
* Deployment Method(snap/docker/sandstorm/mongodb bundle/source):
* Http frontend if any (Caddy, Nginx, Apache, see config examples from Wekan GitHub wiki first):
* Node Version:
* MongoDB Version:
* Wekan only works on newest desktop Firefox/Chromium/Chrome/Edge/Chromium Edge and mobile Chrome. What webbrowser version are you using?

**Problem description**:
- *REQUIRED: Add recorded animated gif about how it works currently, and screenshot mockups how it should work. Use peek to record animgif in Linux https://github.com/phw/peek*
- *Explain steps how to reproduce*
- *In webbrowser, what does show Right Click / Inspect / Console ? Chrome shows more detailed info than Firefox.*
- *If using Snap, what does show command `sudo snap logs wekan.wekan` ? Please anonymize logs.* 
- *If using Docker, what does show command `sudo docker logs wekan-app` ? Please anonymize logs.*
- *If logs are very long, attach them in .zip file*

**Comment by xet7** [comment]
@ufalke 

Yes, someone made subtasks in some strage way https://github.com/wekan/wekan/issues/3868#issuecomment-962439360
